webapp/src/components/propertyValueElement.tsx

```diff
@@ -110,7 +110,7 @@ const PropertyValueElement = (props:Props): JSX.Element => {
                 onCreate={
                     async (newValue, currentValues) => {
                         const option: IPropertyOption = {
-                            id: Utils.createGuid(),
+                            id: Utils.createGuid(''),
```

Comment:
I think the ones in this files are `blocks`

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/155dab31629dc0b9728d6d730187a8f2d6563244